File: examples/spring-test-platform-7/src/test/java/org/camunda/community/process_test_coverage/examples/spring_test/platform7/NonCamundaTest.java
package org.camunda.community.process_test_coverage.examples.spring_test.platform7;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;

@SpringBootTest
public class NonCamundaTest {

    @Test
    public void testSomethingElseAgain() {
        Assertions.assertTrue(true);
    }


}

File: regression-tests/spring-test-platform-7/src/test/kotlin/org/camunda/community/process_test_coverage/spring_test/platform7/NonCamundaTest.kt
package org.camunda.community.process_test_coverage.spring_test.platform7

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.context.annotation.Configuration


@Configuration
class NonCamundaApplication

@SpringBootTest(classes = [NonCamundaApplication::class])
class NonCamundaTest {

    @Test
    fun testSomethingNotCamunda() {
        val testString = "Testing"
        Assertions.assertThat(testString).isNotEmpty()
    }

}